packages/vitest-file-snapshots/src/matchers/register-matchers.test.ts

```diff
@@ -96,19 +107,27 @@ describe("text file matcher", () => {
     expect("4711").toMatchTextFile({ normalizers: [maskNumber] });
   });
 
-  test("uses namingStrategy file by default", () => {
+  test("applies name", () => {
     expect.soft("value1").toMatchTextFile({ name: "value 1" });
     expect.soft("value2").toMatchTextFile({ name: "value 2" });
   });
 
-  test("applies naming strategy fileSuffix", () => {
+  test("when matcher is inverted, throws error", () => {
+    expect(() => expect("value").not.toMatchTextFile()).toThrowError();
+  });
+
+  test("applies custom file path resolver", () => {
     expect("value").toMatchTextFile({
       name: "name",
-      namingStrategy: "fileSuffix",
+      resolveFilePath: testFilePathResolver,
     });
   });
-
-  test("when matcher is inverted, throws error", () => {
-    expect(() => expect("value").not.toMatchTextFile()).toThrowError();
-  });
```

Comment:
Why did you drop this test?

Reply:
The test is still there, but I moved it upwards so it's not between the option-related tests anymore.